_infra/helm/locust/locustfiles/requests.json

```diff
@@ -0,0 +1,32 @@
+{
+    "requests": [
+        {
+            "method": "GET",
+            "url": "/sign-in",
+            "expected_response_text": "Sign in"
+        },
+        {
+            "method": "POST",
+            "url": "/sign-in",
+            "data": {
+                "username": "49900000000@test.com",
+                "password": "password"
+            }
+        },
+        {
+            "method": "GET",
+            "url": "/surveys/todo",
+            "expected_response_text": "Click on the survey name to complete your questionnaire"
+        },
+        {
+            "method": "GET",
+            "url": "/surveys/history",
+            "expected_response_text": ""
```

Comment:
This doesn't seem right, the history will be blank but this will bypass ```if expected_response_text``` in the mixin. It might need extending to include the url now

Reply:
I have added a check to the Mixin where it also makes sure the URL is correct so that it knows its on the correct page.

Reply:
This is now using the basic page title HTML for this initial journey.